packages/processor/test/specimens/composes/external-compose-via-local-compose.css

```diff
@@ -0,0 +1,16 @@
+.ruleWithSingleCompose {
+    composes: red from "../deps/classes.css";
+}
+
+.ruleWithLocalComposes {
+    composes: ruleWithSingleCompose;
+    composes: blue from "../deps/classes.css";
+}
+
+.nextRuleWithLocalComposes {
+    composes: ruleWithSingleCompose;
+}
+
+.nextRuleWithLocalComposes {
```

Comment:
is the duplication of `.nextRuleWithLocalComposes` intended here?

Reply:
It's not necessarily a duplication, but just a continuation of class definition, an alternative notation for:
```css
.nextRuleWithLocalComposes {
    composes: ruleWithSingleCompose;
    composes: blue from "../deps/classes.css";
}
```
I just want to make sure that every case works. 😅